### PR TITLE
fix: hiding isAdmin property

### PR DESCRIPTION
**Pull request by Kif-Fando**

### CODE REVIEW

File: src/users/api/UserDto.ts [hunk 1]
```diff
@@ -10,21 +11,29 @@ export class UserDto {
   @ApiProperty()
   lastName: string;
 
-  @ApiHideProperty()
+  @Exclude()
   isAdmin?: boolean;
 
-  @ApiHideProperty()
```
Comment (derevnjuk) [inline src/users/api/UserDto.ts:16]: Leave `@ApiHideProperty()` for  excluded fields

**Review by derevnjuk — DISMISSED** [review]

File: src/users/api/UserDto.ts (hunk 1 above)
Comment (Kif-Fando) [reply src/users/api/UserDto.ts:16]: +

**Review by derevnjuk — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "@nestjs/swagger": "^4.7.12",
     "axios": "^0.21.2",
     "bcrypt": "^5.0.0",
+    "class-transformer": "^0.4.0",
```
Comment (derevnjuk) [inline package.json:35]: @Kif-Fando when you install a new package, you should commit both package.json and package-lock.json files

Comment (Kif-Fando) [reply package.json:35]: Oh, my fault

Comment (derevnjuk) [reply package.json:35]: Never mind, I have already fixed that

**Pull request merged**